src/auditor/cli.py

```diff
@@ -38,24 +43,66 @@ def cli():
         print(__doc__)
     else:
 
-        # report_dir = args['--save_report']
+        #: Logger that will gather the summary information.
+        summary_logger = logging.getLogger(__name__)
+        summary_logger.setLevel(logging.DEBUG)
 
-        cli_logger = logging.getLogger('auditor')
-        cli_logger.setLevel(logging.DEBUG)
-        detailed_formatter = logging.Formatter(
-            fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
-        )
-        cli_handler = logging.StreamHandler(stream=sys.stdout)
-        cli_handler.setLevel(logging.DEBUG)
-        cli_handler.setFormatter(detailed_formatter)
-        cli_logger.addHandler(cli_handler)
+        if args['spot']:
```

Comment:
**thought**: spot reads to me as a discovery command - almost like a dry run where no changes are made. But you already have a dry option and spot does make changes. 

I wonder if it would be more explicit if the command name was more active like `fix` where the dry option negates the command meaning.

not super important but i think overall useful.

Reply:
I was thinking "spot audit" vs "scheduled audit". Maybe trying to be too clever?

And yes, I think ultimately the 10,000 logic needs to be refactored, which may impact the way the check and fix steps happen and prompt a different cli option.